data/blog/electron-6-0.md

```diff
@@ -0,0 +1,122 @@
+ * dialog.showCertificateTrustDialog() [#17181](https://github.com/electron/electron/pull/17181)
+ * inAppPurchase.getProducts() [#17355](https://github.com/electron/electron/pull/17355)
+ * inAppPurchase.purchaseProduct()[#17355](https://github.com/electron/electron/pull/17355)
+ * netLog.stopLogging() [#16862](https://github.com/electron/electron/pull/16862)
+ * session.clearAuthCache() [#17259](https://github.com/electron/electron/pull/17259)
+ * session.clearCache()  [#17185](https://github.com/electron/electron/pull/17185)
+ * session.clearHostResolverCache() [#17229](https://github.com/electron/electron/pull/17229)
+ * session.clearStorageData() [#17249](https://github.com/electron/electron/pull/17249)
+ * session.getBlobData() [#17303](https://github.com/electron/electron/pull/17303)
+ * session.getCacheSize()  [#17185](https://github.com/electron/electron/pull/17185)
+ * session.resolveProxy() [#17222](https://github.com/electron/electron/pull/17222)
+ * session.setProxy()  [#17222](https://github.com/electron/electron/pull/17222)
+ * webContents.hasServiceWorker() [#16535](https://github.com/electron/electron/pull/16535)
+ * webContents.printToPDF() [#16795](https://github.com/electron/electron/pull/16795)
+ * webContents.savePage() [#16742](https://github.com/electron/electron/pull/16742)
+ * webFrame.executeJavaScript() [#17312](https://github.com/electron/electron/pull/17312)
+ * webFrame.executeJavaScriptInIsolatedWorld() [#17312](https://github.com/electron/electron/pull/17312)
+ * webviewTag.executeJavaScript() [#17312](https://github.com/electron/electron/pull/17312)
+
+These functions now have two forms, synchronous and Promise-based asynchronous:
+ * `dialog.showMessageBox()` [#17298](https://github.com/electron/electron/pull/17298)
+ * `dialog.showOpenDialog()` [#16973](https://github.com/electron/electron/pull/16973)
+ * `dialog.showSaveDialog()` [#17054](https://github.com/electron/electron/pull/17054)
+
+These functions now return Promises:
+ * `app.dock.show()` [#16904](https://github.com/electron/electron/pull/16904)
+
+### `Electron Helper (Render).app` and `Electron Helper (Plugin).app`
+
+There are now more Mac Helper application bundles.
+
+In order to enable the [hardened runtime](https://developer.apple.com/documentation/security/hardened_runtime_entitlements?language=objc), which restricts things like
+writable-executable memory and loading code signed by a different Team
+ID, special code signing entitlements must be granted to the Helper.
+
+To keep the capabilities scoped to the process types that require them, Chromium [added](https://chromium-review.googlesource.com/c/chromium/src/+/1627456) 
+two new variants of the Helper app: one for renderers (`Electron Helper (Render).app`) and one for plugins (`Electron Helper (Plugin).app`).
+
+Currently all three Helpers (the pre-existing `Electron Helper.app` in addition to the two new ones) will be code signed in the same way without
```

Comment:
This block of text is inaccurate at this time, we don't code sign these helpers users do.  Something like this block would probably fit better for Electron.

```
Folks using `electron-osx-sign` to codesign their Electron app
shouldn't have to make any changes to their build logic, if you
are code signing your app with custom scripts you should ensure
that the two new Helper applications are correctly codesigned.
```